### PR TITLE
improve archiving guideance; closes #438

### LINKED ISSUE #438: Improve archival guidance (rename README, have super small README)
Idea by @joelnitta

### PR DESCRIPTION
**Pull request by mpadge**
@joelnitta Please suggest any improvements to this PR. (Sorry, can't request official review from you, as you're outside the org.) Thanks!

### CODE REVIEW

File: maintenance_evolution.Rmd [hunk 1]
```diff
@@ -196,3 +196,19 @@ You don't have to change the tests of deprecated functions until they are made d
 
 Once a function is made defunct, its tests are simply removed.
 
+## Archiving packages {#archivalguidance}
+
+Software generally has a finite lifespan, and packages may eventually need to be archived. Archived packages are moved to a dedicated GitHub organization, [ropensci-archive](https://github.com/ropensci-archive). Prior to archiving, the contents of the README file should be moved to an alternative location (such as "README-OLD.md"), and replaced with minimal contents including the following:
```
Comment (maelle) [inline maintenance_evolution.Rmd:201]:
```suggestion
Software generally has a finite lifespan, and packages may eventually need to be archived. Archived packages are [archived](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories) and moved to a dedicated GitHub organization, [ropensci-archive](https://github.com/ropensci-archive). Prior to archiving, the contents of the README file should be moved to an alternative location (such as "README-OLD.md"), and replaced with minimal contents including the following:
```

File: maintenance_evolution.Rmd [hunk 2]
```diff
@@ -196,3 +196,19 @@ You don't have to change the tests of deprecated functions until they are made d
...
+
+- Title with name of package
+- A repostatus.org badge of "unsupported" for formerly released packages, or "abandoned" for former concept or WIP packages.
```
Comment (maelle) [inline maintenance_evolution.Rmd:204]: Could we add the Markdown blocks to copy-paste here?

File: maintenance_evolution.Rmd [hunk 3]
```diff
@@ -196,3 +196,19 @@ You don't have to change the tests of deprecated functions until they are made d
...
+- One further sentence stating something like, "This package has been archived. The former README is now in README-old", with a link to the old README contents.
+
+An example of a minimal README in an archived package is in [ropensci-archive/monkeylearn](https://github.com/ropensci-archive/monkeylearn/blob/master/README.md). Once the README has been copied elsewhere and reduced to minimal form, the following steps should be followed:
+
+- Close issues with a sentence explaining the situation and linking to this guidance.
```
Comment (maelle) [inline maintenance_evolution.Rmd:209]: could we add a template here too?

File: maintenance_evolution.Rmd [hunk 4]
```diff
@@ -196,3 +196,19 @@ You don't have to change the tests of deprecated functions until they are made d
...
+- Disable issues on GitHub (under repo settings).
```
Comment (maelle) [inline maintenance_evolution.Rmd:210]: I think this is useless if the repo is archived. this was a leftover from when I thought we wouldn't archive repo

File: maintenance_evolution.Rmd [hunk 5]
```diff
@@ -196,3 +196,19 @@ You don't have to change the tests of deprecated functions until they are made d
...
+- Archive the repository on GitHub (also under repo settings).
+- Transfer the repository to [ropensci-archive](https://github.com/ropensci-archive), or request an rOpenSci staff member to transfer it.
```
Comment (maelle) [inline maintenance_evolution.Rmd:212]:
```suggestion
- Transfer the repository to [ropensci-archive](https://github.com/ropensci-archive), or request an [rOpenSci staff member](https://ropensci.org/about/#team) to transfer it (you can email `info@ropensci.org`).
```

**Review by maelle — CHANGES_REQUESTED** [review]
Thank you!

File: maintenance_evolution.Rmd (hunk 2 above)
Comment (mpadge) [reply maintenance_evolution.Rmd]: I did have that, but decided text was neater. But i've just pushed a commit which reverts back to template.

File: maintenance_evolution.Rmd (hunk 3 above)
Comment (mpadge) [reply maintenance_evolution.Rmd]: I've added checkboxes on those points. I that what you meant?

Comment (maelle) [reply maintenance_evolution.Rmd]: no, I mean the Markdown text for the badges

Comment (mpadge) [reply maintenance_evolution.Rmd]: Oh, i did that already, non?

**Pull request merged**